src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +184,23 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    public void verifyInstructorChangesAfterEdit(int instrNum, String newName, String newEmail,
+            Boolean newIsDisplayedToStudents, String newDisplayName, String newRole) {
+        assertEquals(newName, getInstructorName(instrNum));
+        if (!newEmail.isEmpty()) {
+            assertEquals(newEmail, getInstructorEmail(instrNum));
+        }
```

Comment:
fixed in https://github.com/TEAMMATES/teammates/pull/7559/commits/6bc9633b25bfc87bb94b016ba8227f0412d912da